src/stk/molecular/topology_graphs/polymer/linear/linear.py

```diff
@@ -440,8 +440,27 @@ def _get_building_block_vertices(self, building_blocks, vertices):
                 building_block_vertices.get(bb, [])
             )
             building_block_vertices[bb].append(vertex)
+
+        building_block_vertices = self._with_unaligning_vertices(
+            building_block_vertices=building_block_vertices,
+        )
+
         return building_block_vertices
 
+    @staticmethod
+    def _with_unaligning_vertices(building_block_vertices):
+        clone = dict(building_block_vertices)
+        for building_block, vertices in clone.items():
+            # Building blocks with 1 placer, cannot be aligned and
+            # must therefore use an UnaligningVertex.
+            if len(set(building_block.get_placer_ids())) == 1:
+                clone[building_block] = tuple(map(
+                    UnaligningVertex.init_from_vertex,
```

Comment:
I would just do 
```python
tuple(
  UnaligningVertex(
      id=vertex.get_id(),
      position=vertex.get_position(),
      flip=vertex.get_flip(),
  ) for vertex in vertices
)
```

and remove the new `init` method. I think methods should be added fairly rarely and only when they are needed. I think it is not needed in this case.